.gitignore

```diff
@@ -1,15 +1,4 @@
-logs
-*.log
-pids
-*.pid
-*.seed
-lib-cov
-coverage
-.grunt
-.lock-wscript
-build/Release
 node_modules
-.idea
 .vscode
-lib
-package-lock.json
+build
+package-lock.json
```

Comment:
`package-lock.json` shouldn't be in the gitignore, refer to https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

Reply:
I merged before seeing this, will fix asap